src/components/organims/welcome.module.css

```diff
@@ -0,0 +1,138 @@
+body{
+  overflow-x: hidden;
```

Comment:
Hum yes can surely change to it ! 

Did saw some [emotion](https://github.com/getstation/website-v3/blob/master/src/components/atoms/LinkFooter.js#L10) styles so wasn't sure about the right one to choose, I just checked the recommended [providing by Gatsby](https://www.gatsbyjs.org/tutorial/part-two/#css-modules) way to do it but for uniformisation it will be better to have uniq way 👍

So the question is *styled* or *emotion* ? 😅

Reply:
_Untill now, I had mingled styled component and emotion and maybe others. Sorry about that_

From what  see, the choice has already been made for this project: it's `emotion`: i see it in Readme and seems to be used all other. So I guess, it should be `emotion`.